packages/default.nix

```
@@ -24,8 +24,8 @@ rec {
     src = prev.fetchFromGitHub {
       owner = "cuties-social";
       repo = "custom-emojis";
-      rev = "ea8575bfed2c6f259d04f3761a8781dc1ce3387d";
-      hash = "sha256-XROjTzXHCDb/U9SR4McTrbcUSqPGC1eGH9YeVJcRMa8=";
+      rev = "main";
```

```suggestion
      rev = "7bcc7d06cd871e69b2dd666348cf81f23e10a87d";
```

This will cause problems in the future, because `main` will point to commits producing another `hash`.